.github/workflows/applications.yml

```diff
@@ -40,38 +41,14 @@ jobs:
     - name: Get sources
       uses: actions/checkout@v3
 
-    - name: Set up Node.js and restore cache
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.NODE }}
-        cache: 'npm'
-        cache-dependency-path: '**/package-lock.json'
-
-    - name: Get Yarn cache directory path
-      id: yarn-cache-dir-path
-      run: echo "dir=$(yarn cache dir)" >> ${{ matrix.OS != 'windows-latest' && '$GITHUB_OUTPUT ' || '$env:GITHUB_OUTPUT' }}
-
-    - name: Restore Yarn cache
-      uses: actions/cache@v3
-      with:
-        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-        key: ${{ runner.os }}-yarn-${{ matrix.NODE }}-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-yarn-${{ matrix.NODE }}
-          ${{ runner.os }}-yarn-
-
-    - name: Disable taobao registry
-      if: ${{ startsWith( matrix.APPROACH, 'vue') }}
-      run: |
-        echo {\"useTaobaoRegistry\": false} > ${{ env.VUE_CLI_CONFIG_PATH }}
```

Comment:
In the past, Vue CI had many failures when installing npm packages, and the `useTaobaoRegistry` support resolved this issue. Perhaps we shouldn't remove this option?

Reply:
Please @iBat reply

Reply:
This was achieved by pining registry in [packages/devextreme-cli/src/applications/application.vue.js](https://github.com/DevExpress/devextreme-cli/pull/819/files/fe475ab8065ebfe910b2875aecf753a190bc0c6b#diff-16df00d52a46c198d2c7ca0e74aac912be612bbccae48d53042202ba8e452741)
so this step is unnecessary. Btw now it doesn't work.